test/uk/gov/hmrc/cardpaymentfrontend/controllers/EmailAddressControllerSpec.scala

```diff
@@ -110,22 +110,6 @@ class EmailAddressControllerSpec extends ItSpec {
         langToggleText should contain theSameElementsAs List("English", "Newid yr iaith i’r Gymraeg Cymraeg") //checking the visually hidden text, it's simpler
       }
 
-      "render the page with a back link" in {
```

Comment:
We should keep these, no harm in it - if for some reason the layout is changed accidentally, we want to know it breaks our expectations here.